src/explore-education-statistics-admin/src/pages/methodology/createMethodologyPage.tsx

```diff
@@ -0,0 +1,223 @@
+import Link from '@admin/components/Link';
```

Comment:
Pascal-case the file names that export Components (i.e. createMethodolodyPage.tsx => CreateMethodologyPage.tsx) in keeping with the other Components